src/applications/vaos/api/index.js

```diff
@@ -276,7 +276,7 @@ export function getRequestLimits(facilityId, typeOfCareId) {
 export function getClinics(facilityId, typeOfCareId, systemId) {
   let promise;
   if (USE_MOCK_DATA) {
-    if (facilityId.includes('983')) {
+    if (facilityId === '983') {
```

Comment:
This not being exact was messing with the logic for matching clinic ids to past appointments